src/tests/test_gnu_linux.py

```diff
@@ -23,12 +23,8 @@
 
 @pytest.fixture
 def display() -> str:
-    vdisplay = xvfbwrapper.Xvfb(width=WIDTH, height=HEIGHT, colordepth=DEPTH)
-    vdisplay.start()
-    try:
-        yield f":{vdisplay.new_display}"
-    finally:
-        vdisplay.stop()
+    with pyvirtualdisplay.Display(size=(WIDTH, HEIGHT), color_depth=DEPTH) as vdisplay:
```

Comment:
I've used the context manager approach as that's the simplest way of ensuring that `start()` and `stop()` are called automatically.